content/articles/pushserver.rst

```diff
@@ -0,0 +1,184 @@
+:title: pushserver
+:date: 2015-02-09 13:01
+:summary: Settings configuration system based on entry points of setuptools
```

Comment:
@bubenkoff: Looks like this was copy/pasted from another article (should be obvious which one ;-). Can you please add a proper summary here? The [archive of articles on the live site](http://developer.paylogic.com/archives.html) is confusing right now (the summary texts are shown there) because this pull request was already merged and is now live.